pkg/configuration/load_configuration.go

```diff
@@ -0,0 +1,123 @@
+package configuration
+
+import (
+	"context"
+	"os"
+	"strings"
+
+	errs "k8s.io/apimachinery/pkg/api/errors"
+
+	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+// LoadFromSecret retrieves an operator secret and sets a map
+// in order to override default configurations.
+// If no secret is found, then configuration will use defaults.
```

Comment:
Could you please fix the comment? 
this function doesn't set a map and also it doesn't care about default values.
you could clarify here how the default values should be used, what is in the map etc...

Reply:
@MatousJobanek hmm i'm a bit confused, it does set a map based off of what is found in the secret. how would you write this comment in order to be more clear?

Reply:
I was thinking something along these lines:
```suggestion
// LoadFromSecret retrieves an operator secret, loads all keys and values from the secret
// and stores them in a map. This map is then returned by the function.
// The function doesn't take into account any default values - this has to be 
// handled while getting the values in the configuration object. 
```

Reply:
@tinakurian ^

Reply:
sweet! done as part of: d24d418681e1bcca5e857f042cb5d6f07fa08162